src/huggle_ui/reportuser.cpp

```diff
@@ -324,6 +324,7 @@ void ReportUser::OnReportUserTimer()
                 //this->close();
                 delete this;
             }
+            this->close();
```

Comment:
line 318 is useless if we just immediately close this window, the related translation probably too (but double check, maybe we use it somewhere else?)

Reply:
I've added a single shot timer of 2 sec before closing window.